.github/workflows/README.md

```diff
@@ -0,0 +1,155 @@
+# AlphaX Release Process
+These [reusable workflow](https://docs.github.com/en/actions/sharing-automations/reusing-workflows) aim at unifying the release process for all AlphaX packages.
+It automates the creation of a draft release on GitHub, the building of installers for different platforms,
+testing of the installers, the publishing of the package on PyPi and (optionally) the publishing on Docker Hub.
+
+The first part of this Readme explains the release process itself.
+The second part shows how to incorporate this pipeline into your repository.
+The last part explains how to further develop this pipeline.
+
+## Release a new version
+
+### Release a new version
+Note: these instructions assume that the release is done from the `main` branch.
+
+1. Bump the version locally to (e.g. to `X.Y.Z`), e.g. using `bumpversion`, and push this change to `main`.
+2. Manually run the 'Create Draft Release' workflow 
+(the one referencing [this workflow](https://github.com/MannLabs/alphashared/.github/workflows/create_release.yml))
```

Comment:
(nit) I would maybe even drop the bracket with (the one referencing ...). I think the information is not strictly necessary in this place and for me it disrupted a bit the flow of reading as I tried to figure out why the reference was there.